view/frontend/web/template/payment/omise-cc-form.html

```diff
@@ -34,7 +34,26 @@
                    'data-container': getCode() + '-cc-token',
                    },
                    value: omiseCardToken"/>
-            <fieldset data-bind="attr: {class: 'fieldset payment items ccard ' + getCode(), id: 'payment_form_' + getCode()}">
+
+            <!-- ko if: isCustomerHasCard() -->
+                <h4><!-- ko text: 'Select a card you want to proceed' --><!-- /ko --></h4>
+                <ul style="padding-left: 1.3em; margin-bottom: 2em;">
+                    <!-- ko foreach: getCustomerCards() -->
+                        <li style="list-style: none; margin-bottom: 2rem;">
+                            <input name="payment[omise_card]" type="radio"
+                                data-bind="attr: {id: 'Card' + value}, checked: $parent.omiseCard, value: value"/>
```

Comment:
What happens with saved cards that are expired?

Reply:
Looking back through the code, I cannot see any code that is checking this? Are they automatically removed from a user's saved cards at the Omise end? If not, we should really check them and display them as disabled here (probably with an option to remove). Allowing them to go ahead and attempt use them would be a bad user experience as they would end up being told payment failed when we knew that already

Reply:
Good point 👍

Reply:
so... that will be solved in separated PR